.github/dependabot.yml

```diff
@@ -3,11 +3,16 @@ updates:
 - package-ecosystem: gomod
   directory: "/"
   schedule:
-    interval: weekly
+    interval: daily
```

Comment:
Personally, I would prefer to keep this as weekly to limit the daily noise we have to deal with

Reply:
sure!

Reply:
thanks